README.md

```diff
@@ -134,3 +134,15 @@ Shepherd is written in TypeScript, which requires compilation to JavaScript. Whe
 Shepherd currently has minimal test coverage, but we're aiming to improve that with each new PR. Tests are written with Jest and should be named in a `*.test.ts` alongside the file under test. To run the test suite, run `npm run test`.
 
 We use [TSLint](https://github.com/palantir/tslint) to ensure a consistent coding style and to help prevent certain classes of problems. Run `npm run lint` to run the linter, and `npm run fix-lint` to automatically fix applicable problems.
+
+### Semantic Releases
+
+Shepherd leverages [semantic-release](https://github.com/semantic-release/semantic-release) to automate GitHub releases, including:
+- CHANGELOG updates made via [@semantic-release/changelog](https://github.com/semantic-release/changelog)
+- Release Notes compiled via [@semantic-release/release-notes-generator](https://github.com/semantic-release/release-notes-generator)
+- Tagging based on commit message via [@semantic-release/commit-analyzer](https://github.com/semantic-release/commit-analyzer)
+  - Commits that contain `BREAKING CHANGE`, `BREAKING CHANGES`, or `BREAKING` in their body will be considered breaking changes.
+  - Commits with a `docs`, `style`, `refactor`, or `perf` type will be associated with a patch release.
+  - If a commit doesn't match any of the above rules, it will be evaluated against the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js).
+
+  Note: `semantic-release` depends on `GH_TOKEN` secret being to a GitHub token with proper perms. Users need not set this locally.
```

Comment:
I don't know if this relevant information, feel free to ignore if not.

```suggestion
Note: `semantic-release` depends on `GH_TOKEN` secret being to a GitHub token with proper perms. Users need not set this locally. The token was generated and added as a [secret](https://github.com/NerdWalletOSS/shepherd/settings/secrets) by @aorinevo.
```

Also, I didn't think this was meant to be nested within the above bullet, so I removed the initial spaces.